server/data/reviews.js

```diff
@@ -74,30 +75,37 @@ const exportedMethods = {
       product: productId,
     };
     const insertInfo = await reviewCollection.insertOne(newReview);
-    if (insertInfo.insertedCount === 0)
-      throw { message: "Could not add review", code: 500 };
+    if (insertInfo.insertedCount === 0) throw "Could not add review";
 
     let rateall = 0;
     const prodreview = await this.getReviewbyProductId(productId);
+    let arr = [];
     for (let i = 0; i < prodreview.length; i++) {
       rateall = parseInt(prodreview[i].rating) + rateall;
+      arr.push({ _id: prodreview[i]._id });
+    }
+    let averagerate = 0;
+    if (prodreview.length != 0) {
+      averagerate = rateall / prodreview.length;
     }
-    let averagerate = rateall / prodreview.length;
     averagerate = averagerate.toFixed(2);
     const prodCollection = await products();
+
     const updated = {
       rating: averagerate,
+      reviews: arr,
     };
-
     const updatedInfo = await prodCollection.updateOne(
       { _id: productId },
       { $set: updated }
     );
     if (!updatedInfo.matchedCount && !updatedInfo.modifiedCount)
-      throw 'Update rating failed'; 
-  
-    return newReview;
-    },
+      throw "Update rating failed";
+    //Fix to show reviews
+    // Fix to run seed file change to
+    //return newReview;
+    return insertInfo;
```

Comment:
please remove these

Reply:
the comments

Reply:
This is the fix for the seedfile.  
Line 106: Is required in order to run the seedfile to avoid a db_id check. If id is not provided it throws an error while running the seed file.
Line 107 is what is required when adding new reviews via the webapp where we are checking for the ID